wings/data_source/bamboo_relay_local_cluster_order_book_data_source.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
```

Comment:
Local cluster data sources were really only for CoinAlpha's internal use before hummingbot was released. I'd consider any local cluster order book data source to be legacy code by now. So there's no need to have this.

Reply:
Makes sense - the Radar files still references this as well, so you might want to also take this out while you are at it.